generated/nidcpower/nidcpower/session.py

```diff
@@ -3772,20 +3772,21 @@ def _initiate_with_channels(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    # TODO(smooresni): Replace with session lock per GitHub issue [1596](https://github.com/ni/nimi-python/issues/1596)
     def lock(self):
         '''lock
 
-        Obtains a multithread lock on the device session. Before doing so, the
+        Obtains a multi-threaded lock on the instrument driver. Before doing so, the
         software waits until all other execution threads release their locks
-        on the device session.
+        on the driver.
 
-        Other threads may have obtained a lock on this session for the
+        Other threads may have obtained a lock on this driver for the
         following reasons:
 
             -  The application called the lock method.
-            -  A call to NI-DCPower locked the session.
+            -  A call to NI-DCPower locked the driver.
```

Comment:
By the way, this change would be tracked in issue #1596

Reply:
> Hence, the implementation in this PR will lock the entire driver since there is only 1 "global" lock. In other words, only 1 session will be allowed to make driver calls at any given time. Thats the motivation for the wording changes

I think it's better to leave the wording as is (correct), and file an issue that talks of the behavior (incorrect) where the lock is global for Session not for each instance of Session.

Reply:
One less things for us to remember to pull out.

Reply:
Reverted docstrings in e892953d1562389bb3e801b81593e81c24e754de